LabelGenerator.py

```diff
@@ -347,6 +347,8 @@ def draw_resistor_colorcode(
         num_codes: int,
         tolerance_value: Optional[float],
 ) -> None:
+    if value.ohms_val % (10 ** (4 - num_codes)):
```

Comment:
Would you mind adding a comment before this `if`? Like `# Only display resistors that can accurately represent the given value`.